Configure

```diff
@@ -1495,6 +1495,14 @@ if ($target =~ /linux.*-mips/ && !$disabled{asm}
         unshift @{$config{cxxflags}}, $value if $config{CXX};
 }
 
+if ($target =~ /linux.*-loongarch.*/ && !$disabled{asm}) {
+        my $cc = $config{CROSS_COMPILE}.$config{CC};
+        if (!OpenSSL::config::okrun('echo xvadd.w \$xr0,\$xr0,\$xr0',
+                   "$cc -c -x assembler - -o /dev/null 2>/dev/null")) {
+            $disabled{asm} = 'no-asm';
```

Comment:
Doesn't this disable loongarch code which isn't dependent on vector instructions? For example, `CRYPTO_memcmp` and `OPENSSL_cleanse`.

Reply:
Thanks.

@xry111 Do you think that has the same issue? Any suggestions?

https://github.com/openssl/openssl/blob/e1002c84725a64b6a097f3155dc6851b57f7ba8e/util/perl/OpenSSL/config.pm#L687-L696

Reply:
> Thanks.
> 
> @xry111 Do you think that has the same issue? Any suggestions?
> 
> https://github.com/openssl/openssl/blob/e1002c84725a64b6a097f3155dc6851b57f7ba8e/util/perl/OpenSSL/config.pm#L687-L696

Remove the detection here.  I added it at the wrong place :(.